app/helpers/record_helper.rb

```diff
@@ -98,6 +98,48 @@ def access_type(metadata)
     access_right.first['description']
   end
 
+  def issued_date(dates)
+    return if dates.blank? || dates&.none? { |date| date['kind'] == 'Issued' }
+
+    # If there is more than one date issued, take the first one.
+    dates.select { |date| date['kind'] == 'Issued' }&.uniq&.first['value']
+  end
+
+  def coverage_date(dates)
+    return if dates.blank? || dates&.none? { |date| date['kind'] == 'Coverage' }
+
+    # If there is more than one coverage date, take the first one.
```

Comment:
I'm not sure this is the behavior we want. I feel we should display multiple dates in this situation, either as comma separate or as a bulleted list.

Reply:
What if I told you this is common?

Reply:
Neat. I think this is a good example we may need to bring to DataEng and/or UX to decide if we want to change the data mappings/better understand the data mappings/better understand what we want to do on the UI side.

For now, if you want to build in duplicate detection logic in the UI I could get behind it. I'm reluctant to just assume all dates are duplicate (without more insight from DataEng as to how these mappings are created at least).

Reply:
Yeah, I generally don't like putting that kind of logic in UI, but it seems like a reasonable solution in this case.

Reply:
I feel like it's still worth a convo between EngX/DataEng at some point to help us better understand all the assumptions about dates. But for now this looks good to me.